File: pc-square-io-separator-java/src/main/java/com/mj/pc/square/io/separator/java/exception/LoadInputFileException.java
package com.mj.pc.square.io.separator.java.exception;

/**
 * @author Montaser Sbaih
 * @version 1.0
 * @email montaser.jjs@gmail.com
 * @phone +962-786258874
 * @since 29-10-2022
 */

public final class LoadInputFileException extends ExecutionException {

    @Override
    protected String getExceptionKey() {
        return "exception.file.input.load";
    }
}

File: pc-square-io-separator-kotlin/src/main/kotlin/com/mj/pc/square/io/separator/kotlin/exception/LoadInputFileException.kt
package com.mj.pc.square.io.separator.kotlin.exception

/**
 * @author Montaser Sbaih
 * @version 1.0
 * @email montaser.jjs@gmail.com
 * @phone +962-786258874
 * @since 29-10-2022
 */

class LoadInputFileException : ExecutionException() {

    override val exceptionKey: String
        get() = "exception.file.input.load"
}